db/migrate/20190716125356_add_fields_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToUsers < ActiveRecord::Migration
+  def change
+    add_reference :users, :company, index: true
+    add_column :users, :f_name, :string
```

Comment:
user proper name for fields f_name -> first_name

Reply:
Fixed.